tests/nonbonded/test_nonbonded_jit.py

```diff
@@ -2,23 +2,22 @@
 import jax.numpy as jnp
 import numpy as np
 
-from timemachine.potentials import generic
+from timemachine.potentials import Nonbonded
 
 
-def test_generic_nonbonded_jittable(rng: np.random.Generator):
+def test_nonbonded_reference_jittable(rng: np.random.Generator):
 
     N = 30
 
-    potential = generic.Nonbonded(
+    U_ref = Nonbonded(
         N,
         exclusion_idxs=jnp.zeros((0,)),
         scale_factors=jnp.zeros((0, 1)),
         beta=1.0,
         cutoff=0.1,
     )
 
-    U_ref = potential.to_reference()
-    U_ref_jit = jax.jit(U_ref)
+    U_ref_jit = jax.jit(U_ref.__call__)
```

Comment:
Slightly unfortunate; `jit` does not seem to work on `U_ref` by itself (one workaround is to add explicit `__call__`)